### DISCUSSION TITLE
[REQ] Let Intel products' owners join the NAM revolution

### DISCUSSION THREAD

**Discussion by MarcoRavich** [intel/openvino-plugins-ai-audacity#281] [Ideas]
Since seems it isn't still mentioned here, it would be great to have an OpenVINO version of the (trendy) glorious @sdatkinson's [NAM: Neural Amp Modeler](https://github.com/sdatkinson/neural-amp-modeler#readme) and @honkkis's [NAM HD: Neural Amp Modeler on Steroids](https://github.com/honkkis/neural-amp-modeler-HD#readme) version for Intel hardware owners, as crossrequested in [this issue](https://github.com/sdatkinson/neural-amp-modeler/issues/455) too.

**Comment by MarcoRavich** [comment]
A2 is coming (introduction video by Leo Gibson):
[![Leo Gibson|The NAM A2 could become the industry standard for amp profiling!)](https://img.youtube.com/vi/NsZL5f-v6rY/maxresdefault.jpg)](https://youtu.be/NsZL5f-v6rY "Watch The NAM A2 could become the industry standard for amp profiling by Leo Gibson")

**Comment by RyanMetcalfeInt8** [comment]
Thanks @MarcoRavich -- thanks for sharing. What do you think a NAM feature would look like for these plugins? Apologies in advance, I'm not very familiar with the use case. Are there a set of pre-trained models available? Or would it require the user to feed in some data & train a model on the fly?

**Reply by MarcoRavich** [reply]
> What do you think a NAM feature would look like for these plugins? 
>
Well, it (maybe ?) would be better to have a separate plugin like:
- @GuitarML's [SmartGuitarAmp](https://github.com/GuitarML/SmartGuitarAmp#readme)
- @brummer10's [NeuralRack](https://github.com/brummer10/NeuralRack#readme);
- @Tr3m's [NAM JUCE](https://github.com/Tr3m/nam-juce#readme);

> Are there a set of pre-trained models available?
> Or would it require the user to feed in some data & train a model on the fly?
>
Both: there are COMMUNITIES of pretrained "models" (but the term "profiles" fits better here) like [TONE3000](https://www.tone3000.com/) but users can train their own too.

Anyway, due to the OpenVINO nature, at the moment it's probably more effective to port (or patch ?) inferencing engines such as:
- @sdatkinson's [NeuralAmpModelerCore](https://github.com/sdatkinson/NeuralAmpModelerCore#readme);
- @mikeoliphant's [NeuralAudio](https://github.com/mikeoliphant/NeuralAudio#readme);
- @jatinchowdhury18's [RTNeural](https://github.com/jatinchowdhury18/RTNeural#readme).

Hope this helps to clarify a bit...
